docs/topics/AutoType.adoc

```diff
@@ -31,26 +31,9 @@ To configure Auto-Type sequences for your entries, perform the following steps:
 .Auto-Type entry sequences
 image::autotype_entry_sequences.png[]
 
-2. _(Optional)_ Define a custom auto-type sequence for each window title match by selecting the _Use specific sequence for this association_ checkbox. Sequence action codes and field placeholders are detailed in the following table. A complete list of supported actions and placeholders can be found at https://keepass.info/help/base/autotype.html#autoseq[KeePass Auto-Type Action Codes] and https://keepass.info/help/base/placeholders.html[KeePass Placeholders]. Action codes and placeholders are not case sensitive.
+2. _(Optional)_ Define a custom auto-type sequence for each window title match by selecting the _Use specific sequence for this association_ checkbox. Sequence action codes and field placeholders are detailed in the following table. A complete list of supported actions and placeholders can be found at https://keepass.info/help/base/autotype.html#autoseq[KeePass Auto-Type Action Codes, window=_blank] and https://keepass.info/help/base/placeholders.html[KeePass Placeholders, window=_blank]. Action codes and placeholders are not case sensitive.

```

Comment:
Why reinvent the documentation wheel? There are a ton of useless placeholders that are defined and not relevant to 99.9% of users. I don't want to clutter our documentation with those, let them live in KeePass' documentation.

Reply:
The documentation should be self-contained. It could go into an extended section or something, but our documentation should include everything we implement, useless or not.

Reply:
Indeed, we can't skimp on documentation and we can't assume an internet connection is available.

Reply:
OK, I added a "References" section to the UserGuide and moved all the extensive, technical bits into this section. I removed the external links to keepass.info and replaced with internal links within the UserGuide.